css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+	display: none;
+}
+
+.yoast .tablenav.top #posts-filter .tablenav-pages {
+	display: block;
+}
+
+.yoast .tablenav .yoast-button {
+	color: var(--color-dark);
+}
+
+.yoast .tablenav .search-box{
+	float: right;
+}
+
+.yoast .tablenav .search-box div {
+	float: left;
+	width: 200px;
+	margin-right: 8px;
+}
+
+.yoast .tablenav .displaying-num {
+	margin-right: 22px;
+}
+
+.yoast .tablenav .paging-input {
+	margin-right: 6px;
+}
+
+.yoast .column-actions {
+	text-align: right;
+	width: 200px;
+}
+
+.yoast .tablenav .tablenav-pages .button,
+.yoast .tablenav .tablenav-pages .button.disabled,
+.yoast .tablenav .tablenav-pages .tablenav-pages-navspan {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
+	border-radius: 4px;
+	transition: background-color 150ms ease-out 0s;
+	cursor: pointer;
+	padding: 10px 13px 12px;
+	border: 1px solid rgba(0,0,0, 0.2)!important;
+	box-shadow: inset 0 -2px 0 rgba(0, 0, 0, 0.1)!important;
+	color: var(--color-dark)!important;
+	line-height: 16px;
+	margin-right: 6px;
+}
+
+.wp-core-ui.yoast .tablenav .tablenav-pages .button.disabled {
+	opacity: 0.5;
+}
+
+.yoast .tablenav-pages .current-page {
+	font-size: var(--font-size-default);
+	padding: 5px 8px;
+	background: white;
+	border: var(--border-default);
+	box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.1);
+	box-sizing: border-box;
+	border-radius: 0px;
+	margin-right: 6px;
+}
+
+/* The following CSS is specific for the bulk-editor tables on the tool pages. */
+
+.yoast select#post-type-filter-title,
+.yoast select#post-type-filter-description {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    font-size: var(--font-size-default);
+    padding: 6px 40px 6px 8px;
+    background-color: white;
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 8"><path d="M1.4 0L6 4.6 10.6 0 12 1.4 6 7.5 0 1.4z"/></svg>');
+    background-position: right 15px center;
+    background-repeat: no-repeat;
+    background-size: 10px auto;
+    border-radius: 0;
+    border: var(--border-default);
+    box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.1);
+    position: relative;
+    max-width: 300px;
+    display: block;
+	min-height: 2.8em;
+	float:left;
+}
+
+.yoast .wpseo_bulk_titles tr th:first-child,
+.yoast .wpseo_bulk_descriptions tr th:first-child{
+	padding-left:16px;
+}
+
+.yoast .wpseo_bulk_titles input,
+.yoast .wpseo_bulk_descriptions textarea {
+	width: 100%;
+	font-size: var(--font-size-default);
+	padding: 8px;
+	background: white;
+	border: var(--border-default);
+	box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.1);
+	box-sizing: border-box;
+	border-radius: 0;
+}
+
+.yoast .wpseo_bulk_titles .row-actions,
+.yoast .wpseo_bulk_descriptions .row-actions {
+	display: none;
+}
+
+.yoast .wpseo_bulk_titles input {
+	height: 40px;
+}
+
+.yoast .yoast-table__container {
+	overflow-x: scroll;
+	width: 100%;
+}
+
+.yoast .yoast-table__container table {
+	min-width: 1500px;
```

Comment:
What is the reasoning behind the choice of this value?

Reply:
1500px is the minimum width the table has to be without content overlapping.

Reply:
Sorry but the space taken by the content can't be predicted, due for example to translations and many other factors. 1500 still makes no great sense to me.